protected/controllers/CsvController.php

```diff
@@ -216,6 +216,7 @@ private function createNewPatient($patient)
         $new_patient->contact_id = $contact->id;
         $new_patient->hos_num = !empty($patient['CERA_number']) ? $patient['CERA_number'] : Patient::getNextCeraNumber();
 
+        $new_patient->setScenario('other_register');
```

Comment:
Shouldn't this be set depending on the value of the 'patient_source' column?

Reply:
Ideally it should. Do we capture the patient source in the CSV?